sdcm/remote.py

```diff
@@ -141,11 +141,13 @@ def _create_connection(self):
         return Connection(host=self.hostname, user=self.user)
 
     def run(self, cmd, timeout=300, ignore_status=False,  # pylint: disable=too-many-arguments
-            connect_timeout=300, verbose=True, log_file=None, retry=0):
+            connect_timeout=300, verbose=True, log_file=None, retry=0, watchers=None, new_session=False):
 
-        watchers = []
+        watchers = watchers if watchers else []
```

Comment:
wasn't the same to use in the function signature ```watchers=[]``` instead of setting it to None and then ```if not watchers``` set it to an empty list?

Reply:
oh no, did this mistake once and :boom:

putting a mutable object in the function signature mean it would be create only once
and we append to this list...

we had a bug exactly there with this parameter that multiple our logs by x100 and even more

Reply:
thanks